index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="/quiz.css">
```

Comment:
This just needs to be changed to reflect the name of your css file, and you can leave off the `/`.